sickbeard/helpers.py

```diff
@@ -595,8 +593,8 @@ def moveAndSymlinkFile(srcFile, destFile):
         ek(shutil.move, srcFile, destFile)
         fixSetGroupID(destFile)
         ek(symlink, destFile, srcFile)
-    except Exception:
-        logger.log(u"Failed to create symlink of " + srcFile + " at " + destFile + ". Copying instead", logger.ERROR)
+    except Exception as e:
+        logger.log(u"Failed to create hardlink of %s at %s. Error: %r. Copying instead" % (srcFile, destFile, ex(e)),logger.WARNING)
```

Comment:
point is there is no reason to make 60 changes right now at one time. Copy and paste doesnt always work.

We have broken stuff to fix rather than guessing about if you should change a log line.